libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/MapboxSpeechPlayer.java

```diff
@@ -69,18 +73,24 @@ private void setupCaches(Context context) {
   /**
    * Plays the specified text instruction using MapboxSpeech API, defaulting to SSML input type
    *
-   * @param instruction voice instruction to be synthesized and played
+   * @param speechAnnouncement with voice instruction to be synthesized and played
    */
   @Override
-  public void play(String instruction) {
-    play(instruction, SSML_TEXT_TYPE);
+  public void play(SpeechAnnouncement speechAnnouncement) {
+    this.announcement = speechAnnouncement;
+    String ssmlAnnouncement = announcement.ssmlAnnouncememnt();
+    if (ssmlAnnouncement != null) {
```

Comment:
why would `ssmlAnnouncement` be null?

Reply:
Using `VoiceInstructionMilestone` it wouldn't ever be `null`, but if someone uses `NavigationSpeechPlayer` outside of the turn-by-turn UI, they could choose to not include this.